### DISCUSSION TITLE
Are there plans to offer equivalent functionality to 'npm pkg set'?

### DISCUSSION THREAD

**Discussion by AlexJeffcott** [pnpm/pnpm#5025] [Q&A] [answered]
I really enjoy the ability to easily edit my package.json through the command line and npm has made improvements over `npm set-scripts` with `npm pkg set`. This allows easy scripting and documenting of set up.
```shell
pnpm init
npm pkg set author="Alex Jeffcott"
npm pkg set license="MIT"
npm pkg set scripts.test="vitest"
npm pkg set scripts.start="vite"
```

Are there plans to add this to the wonderful pnpm?

**Comment by zkochan** [comment, marked as answer]
Not right now. Just use npm CLI for this.
